charts/premium/traefik/values.yaml

```diff
@@ -120,6 +120,7 @@ metrics:
   main:
     enabled: true
     type: servicemonitor
+    addInternals: true
```

Comment:
We dont want people to randomly add things to sections stemming from common.
There is literally no reason to confuse people, as this makes it look like this is something that can be found int he common docs.

Use a non-common root structure such as .Values.traefik or .Values.whatever.

Reply:
thanks, i'll figure out a new non-common section

Reply:
I see the same result is also achievable using "globalArguments". Do you prefer to have a dedicated section or to close this PR ?

Reply:
Feel free to do whatever with it :)